everett/manager.py

```diff
@@ -240,6 +241,45 @@ def __repr__(self):
         return '<ListOf(%s)>' % qualname(self.sub_parser)
 
 
+class ExistingFile(object):
+    """A parser for files that are expected to already exist"""
```

Comment:
Everett uses autodoc heavily, so at some point, we're going to want to flesh out this docstring and provide examples of usage.